fe/src/app/service/verify/success/page.tsx

```diff
@@ -0,0 +1,18 @@
+import { getSession } from "@auth0/nextjs-auth0";
```

Comment:
Do we really need to have two separate pages for `verify` and `success`? I think we could have it all in one page, just would need to have an endpoint to check if response document is already provided, so we would display a message of `success` page. Give your thoughts on this, maybe I am missing something.

Reply:
I tried this approach you described earlier, but then main page should be client component and it would need to store state.

Then we couldn't use `getSession` hook since it doesn't work in client component. Ok, no big deal, we make it server component. So now our structure looks like this:

**Page (parent, client):
--- Success component (child, server)
--- PdfDropzone (child, client)**

Guess what, when changing the state if success in pdf upload or not and dynamically rendering components using ternary, next js won't switch between rendering client component and server component, for some reason next router error would pop up in console. 

So the solution is separate route... Maybe I am just inexprienced in next js, or it is just trash on dynamic rendering between client and server :D.